examples/platforms/nrf528xx/src/radio.c

```diff
@@ -660,6 +694,7 @@ otError otPlatRadioEnergyScan(otInstance *aInstance, uint8_t aScanChannel, uint1
     return OT_ERROR_NONE;
 }
 
+#if !OPENTHREAD_CONFIG_ENABLE_PLATFORM_POWER_CUSTOM_SOURCE
 otError otPlatRadioGetTransmitPower(otInstance *aInstance, int8_t *aPower)
```

Comment:
These APIs are required by OpenThread and shouldn't be hidden behind the macro.

Reply:
If the user defines the macro `OPENTHREAD_CONFIG_ENABLE_PLATFORM_FEM_CUSTOM_SOURCE `,  he should implement the function `otPlatRadioGetTransmitPower ()` by himself, just like the function `otPlatRadioGetIeeeEui64()`. Thoughts?